core/server/data/schema/schema.js

```diff
@@ -368,5 +368,15 @@ module.exports = {
         mobiledoc: {type: 'text', maxlength: 1000000000, fieldtype: 'long', nullable: true},
         created_at_ts: {type: 'bigInteger', nullable: false},
         created_at: {type: 'dateTime', nullable: false}
+    },
+    members: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        email: {type: 'string', maxlength: 191, nullable: false, unique: true, validations: {isEmail: true}},
+        name: {type: 'string', maxlength: 191, nullable: false},
+        password: {type: 'string', maxlength: 191, nullable: true},
```

Comment:
This column is nullable to support third party auth in future. e.g. sign in with google

Reply:
Is there a reason that the pwd length is 191? 🤔

Reply:
It was the most suitable string size that we use for columns - as I didn't think we were supposed to deviate from them.  I can set it to the exact size of the bcrypt output if that's preferred?

Reply:
Yeah totally - I was just going from this: https://github.com/TryGhost/Ghost/blob/master/core/server/data/schema/schema.js#L1-L9

I would much prefer explicit sizes - but we haven't done that recently for things like tokens. Will update this tomorrow tho